test/plugin_test.py

```diff
@@ -1473,6 +1475,96 @@ def test_invalid_uri(self):
         except lsm.LsmError as le:
             self.assertTrue(le.code == lsm.ErrorNumber.INVALID_ARGUMENT)
 
+    def test_battery_list(self):
+        for s in self.systems:
+            cap = self.c.capabilities(s)
+            if supported(cap, [Cap.BATTERIES]):
+                self.c.batteries()
+
+    def test_volume_cache_info(self):
+        flag_tested = False
+        flag_created = False
+        for s in self.systems:
+            cap = self.c.capabilities(s)
+            if supported(cap, [Cap.VOLUME_CACHE_INFO]):
```

Comment:
You should check for volume create and volume delete capability too, if you have no volumes